### PR TITLE
🚀 Progressive web app

### PR DESCRIPTION
**Pull request by anna-wro**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -34,3 +34,15 @@ yarn-error.log*
...
 # vercel
 .vercel
+
+# PWA files
+**/public/precache.*.*.js
+**/public/sw.js
+**/public/workbox-*.js
+**/public/worker-*.js
+**/public/fallback-*.js
+**/public/precache.*.*.js.map
```
Comment (k1c14k) [inline .gitignore:44]: do we need a map file?

File: src/pages/index.tsx [hunk 2]
```diff
@@ -19,7 +19,7 @@ export function getStaticProps() {
     .map(filePath => {
       const prayerData = getPrayerDataFromFilename({ filePath });
       const source = fs.readFileSync(path.join(PRAYERS_PATH, filePath));
-      const { data, content } = matter(source);
+      const { content, data } = matter(source);
```
Comment (k1c14k) [inline src/pages/index.tsx:22]: do we have an unit test for this change?

Comment (anna-wro) [reply src/pages/index.tsx:22]: in destructuring objects, it doesn't matter

try
```
const { cat, dog } = { cat: "Olek", dog: "Reksio" };
```
and 
```
const { dog, cat } = { cat: "Olek", dog: "Reksio" };
```
Both will give you the same values for the cat.

Comment (anna-wro) [reply src/pages/index.tsx:22]: I used this change to trigger a new deploy, will undo this soon

File: .gitignore (hunk 1 above)
Comment (anna-wro) [reply .gitignore:44]: It's generated automatically.

File: .gitignore [hunk 3]
```diff
@@ -34,3 +34,15 @@ yarn-error.log*
 
 # vercel
 .vercel
+
+# PWA files
```
Comment (anna-wro) [inline .gitignore:38]: I took the example from 
https://blog.usman-s.me/how-to-make-a-nextjs-app-a-pwa-with-offline-support

**Review by k1c14k — APPROVED** [review]

**Pull request merged**